index.html

```diff
@@ -62,21 +60,20 @@ <h4 id="beer-brewery"></h4>
             <div class="form-group col-md-4">
                 <form class="form-horizontal">
                     <fieldset>
-                        <legend class="text-center">Enter Your Beer</legend>
+                        <legend class="text-center col-xs-12">Enter Your Beer</legend>
                         <div class="form-group">
-                            <div class="col-lg-12">
-                                <input type="text" class="form-control" id="input-beer" placeholder="Goose Island Green Line Pale Ale">
+                            <div class="col-xs-12">
+                                <input type="text" class="form-control" id="input-beer" placeholder="Enter beer here">
```

Comment:
You mean as just a placeholder? I didn't actually dismantle the dynamic placeholder logic. I'll ask adam to do a PR to take care of that.